notes.txt

```diff
@@ -0,0 +1,43 @@
+Need to transfer sessions when switching locales as a logged in editor, even
```

Comment:
Are you sure you want to commit this? Are there other P'unk repos with a `notes.txt` file that contains documentation that we don't want in the `README`?